tests/flow/test_graph_rename.py

```diff
@@ -0,0 +1,33 @@
+from RLTest import Env
+from redisgraph import Graph
+
+class test_graph_rename():
+    def __init__(self):
+        self.env = Env()
+        self.redis_con = self.env.getConnection()
+    
+    def test01_dump_rename(self):
+        self.redis_con.flushall()
+
+        graph_name = "x"
+        redis_graph = Graph(graph_name, self.redis_con)
+        
+        # Create node
+        result = redis_graph.query("CREATE (:L)")
+        self.env.assertEquals(result.nodes_created, 1)
+        
+        # Dump
+        data = self.redis_con.dump(graph_name)
+
+        self.redis_con.flushall()
```

Comment:
do not delete "old" graph, keep it!
modify the following create query to create a different node, say: `CREATE (a:V {v:1})`
then validate using `MATCH (a:V) RETURN a.v` on both the original graph `x` and the restored one `y`